### PR TITLE
4.3.2. Add fields to allow the collection of more than one marital partner in social form

### PR DESCRIPTION
**Pull request by ynurmahomed**

### CODE REVIEW

**Comment by ynurmahomed** [comment]
[4.3.2. Add fields to allow the collection of more than one marital partner in social form](https://trello.com/c/ICah3S9i/40-432-add-fields-to-allow-the-collection-of-more-than-one-marital-partner-in-social-form)

File: app/social/components/sexualPartners.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<div class="marital-partners">
+
+  <div class="alert alert-info" ng-if="!vm.checkedIn">
+    {{'CHECKIN_NOT_CHECKED_IN_TODAY'|translate}}
+  </div>
```
Comment (valvijo) [inline app/social/components/sexualPartners.html]: Better if we show consistency here:
image

File: app/social/components/sexualPartnerForm.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<form class="row" name="form" ng-submit="vm.submit(form);" novalidate>
```
Comment (valvijo) [inline app/social/components/sexualPartnerForm.html]: Better if we show consistency of design:
image
and
image
 
With one save button in the end (Next = Save).

File: app/social/sexualPartnersService.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  /* @ngInject */
+  function sexualPartnersService($q, $log, conceptService, encounterService, observationsService, visitService) {
+
+    var SEXUAL_PARTNER_INFORMATION_CONCEPT_UUID = '4cd975c4-56b8-478b-a528-d8ffb9ecd200';
+
+    var PARTNERS_NAME_CONCEPT_UUID = 'bc39b981-bde2-4486-b1b8-8cef5a3233e3';
+
+    var RELATIONSHIP_TO_PATIENT_CONCEPT_UUID = 'e13e172c-9f49-4bd0-a976-b0c167f47918';
+
+    var HIV_STATUS_CONCEPT_UUID = 'e1db0cf0-1d5f-11e0-b929-000c29ad1d07';
```
[the remark this answers is not in the document]
Comment (ynurmahomed) [reply app/social/sexualPartnersService.js]: Cant we do that when it's actually necessary to reuse?

**Comment by ynurmahomed** [comment]
@valvijo

**Comment by valvijo** [comment]
Looks great!

**Pull request merged**